<!--#include file="_header.html" -->
<!--#include file="_bodyhead.html" -->

<h2>Lab Introduction</h2>

<img src="chick-in-shell.jpg" align="right" alt="cute chick in shell">

You have reached the lab for Genomics, Evolution, and Development
(GED) at Michigan State U.  This lab is run by Dr. C. Titus Brown, who
is an assistant professor in the departments of
<a href="http://www.cse.msu.edu/">Computer Science and
Engineering (CSE)</a> and <a href="http://www.mmg.msu.edu/">Microbiology and Molecular Genetics (MMG)</a>
at <a href="http://www.msu.edu/">MSU</a>.
In addition to
<a href='interests.html'>his research interests</a>, he <a href='courses/'>teaches a number of
courses</a>.

<p>

Broadly speaking, we are interested in the mechanisms of development:
how embryos develop, how genomes encode the developmental process, and
how the developmental process has evolved.  Our model organism of
choice is the chick, a particularly accessible vertebrate embryo, and
our primary interest is in the development and evolution of the neural
crest, a fascinating tissue that contributes to a variety of adult
vertebrate structures.  By studying the interactions between
regulatory genes and how they progressively specify tissues, we can
begin to understand the way in which neural crest arises in
development; and, by comparing and contrasting chick development with
zebrafish, frog, mouse, lamprey, and invertebrate chordates such as
amphioxus and ascidians, we can gain insight into the evolutionary
mechanisms by which these structures appeared.

<p>

Our approach combines hypothesis-driven single gene experiments with
expertise in large-scale data integration and gene network model
building.  This combination of "wet" experimental work and "dry"
computational integration is well suited to investigating the large
networks of interconnected molecules involved in neural crest
development.

<p>

The development and extension of data analysis tools and toolkits is
an increasingly large interest of the lab.  We are in dire need
of advanced computational tools for modern biology -- tools for
analyzing data from whole genome sequencing and sequence annotation,
microarrays, ChIP-chip data, massively parallel sequencing, regulatory
genomics, third-party databases, and metagenomic sequences.  As part
of our research we are pushing the interface between investigative
tools and biological science, which will lead to new types of
questions and (hopefully!)  answers to old questions.  In connection
with this research, we are very interested in the process of
developing maintainable and extensible software.

<!--#include file="_bodyfoot.html" -->
