<!--#include file="_header.html" -->
<!--#include file="_bodyhead.html" -->

<h2>Lab Interests</h2>

<p>

The lab interests are diverse, yet there is a common goal: we seek to
intertwingle computation with experiment in order to improve our
understanding of biology.  We intersect with a number of fields, both
new and old, including developmental biology, molecular biology,
bioinformatics, regulatory genomics, metagenomics, and next-generation
sequencing data.

<p>

<h4>Grant proposals and formal writeups:</h4>
<ul>
<li> <a href='downloads/2011-reappointment.pdf'>2011 / Reappointment</a></li>
<p>
"Dear MSU, please do not fire me.  I will be good, I promise."  A writeup
of what I have accomplished and what I hope to accomplish as a professor, and
why MSU should keep me around.
<p>
(They believed me; I'm hired through 2015.)
<p>
<li> <a href='downloads/2010-ngs-course-nih-r25.pdf'>2010 / Next-gen course, NIH R25</a> </li>
<p>
Our not-yet-rejected / in-review grant proposal to support <A href='http://bioinformatics.msu.edu/ngs-summer-course-2011'>our summer course</a> on next-gen sequence analysis.
<p>
<li> <a href='downloads/2009-usda-vertex.pdf'>2009 / Web tools for next-gen sequence analysis</a></li>
<p>
Our (accepted) grant proposal to build an easy-to-use Web interface for
next-gen sequence analysis. (USDA NIFA)
<p>
<li> <a href='downloads/cartwheel-nih-sep2007-short.pdf'>2007 / Cartwheel</a>
<p>
A (rejected) grant
proposal on the need for easy-to-use Web interfaces for comparative sequence
analysis.  (NIH R01)
</ul>

<h4>A brief write-up of some of our interests</h4>

<ul>

<li> Neural crest specification in early vertebrate embryogenesis.</li>
<p>
The neural crest is an important developmental tissue that is
specified early on in embryogenesis.  We seek to understand the
molecular events underlying this specification, using both molecular
and embryological techniques, and integrate it with sequence data to
produce a gene network model for neural crest specification.  (This
is our primary experimental focus.)
</p>
<li> Next-generation sequence analysis.</li>
<p>
Advances in sequencing have led to a data tsunami, necessitating
advances in algorithms, approaches, and training.  We are working on
advanced techniques in de Bruijn graph assembly and k-mer-based data
set filtering, to apply de novo assembly approaches in metagenomics,
mRNAseq, and environmental genomes; one particular emphasis is in
adapting de novo assembly to work well within the constraints of the
Amazon cloud.  We also
teach <a href='http://bioinformatics.msu.edu/ngs-summer-course-2011'>a
yearly summer course</a>.  (Collaborations
with <a href='https://www.msu.edu/~liweim/lilab/'>Li
lab</a>, <a href='http://www.ars.usda.gov/pandp/people/people.htm?personid=980'>Cheng
lab</a>, and many others.)  This is our primary computational focus.
<p>
<li> Metagenomics.</li>
<p>
Most (99% or more) of microbes cannot be cultured in the lab, yet many
microbes play important environmental or medical roles.  Metagenomics
seeks to understand microbial communities by making use of new
sequencing technologies to sequence unculturable organisms, yet this
data cannot be easily grokked.  We are interested in computational
ways to analyze and understand metagenomic data.  (Collaborations with
Schmidt, Tiedje labs.)
</p>
<li> Evolutionary developmental biology.</li>
<p>
The Molgulid ascidians are a fascinating group of invertebrate
chordates that have repeatedly and independently lost their larval
tails, which is otherwise rare in the ascidians.  We seek to
understand the genomic modifications that have made this possible, and
build a detailed molecular understanding of the tail development
network in Molgulids.  We are applying modern sequencing techniques to
investigate gene expression in <i>M. oculata</i>
(tailed), <i>M. occulta</i> (tailed), and hybrids between the two.
(Collaboration
with <a href='http://faculty.washington.edu/bjswalla/'>Swalla lab</a>
at U. Washington, Seattle.
<p>
<li> Animal regulatory genomics.</li>
<p>
Much of early development is "hard wired" in the genome's regulatory
elements.  Finding and analyzing regulatory elements in animal genomes
can be very difficult, given the large size and complexity of these
genomes.  We are building tools to help detect, visualize and analyze
regulatory elements.
</p>
<li> Microbial regulatory genomics.</li>
<p>
Gene regulation is an important part of microbial physiology, yet we
still know relatively little about finding and studying regulatory
elements computationally in microbes.  The smaller size of microbial
genomes makes locating transcription factor binding sites easier, but
validation still requires difficult experiments.  We are particularly
interested in techniques for testing the internal consistency of binding
site predictions.
</p>
<li> Evolutionary sequence signatures.</li>
<p>
As genomes evolve, different selection pressures are brought to bear
upon the various functional elements.  These functional elements can
often be recognized computationally from the signature left by
evolution.  We are interested in using such signatures to
computationally find novel genes and regulatory elements.
</p>
<li> Software engineering methodologies.</li>
<p>
A critical component of modern biologiy is digesting and integrating
data from multiple sources, both local (e.g. lab data, collaborator
data) and remote (NCBI, ENSEMBL, UCSC, other genome databases).  This
research depends on extensive re-use and maintenance of old software
tools as well as development of new software tools, a traditional
challenge of software engineering.
</p>
</ul>

Studying many topics means many colleagues and many collaborators.
Come by and <A href="contact.html">chat</a> today!

<!--#include file="_bodyfoot.html" -->
