<!--#include file="_header.html" -->
<!--#include file="_bodyhead.html" -->

<h2>Lab Interests</h2>

<p>

The lab interests are diverse, yet there is a common goal: we seek to
intertwingle computation with experiment in order to improve our
understanding of biology.  We intersect with a number of fields, both
new and old, including developmental biology, molecular biology,
bioinformatics, regulatory genomics, and metagenomics.

<p>

(You can also <a href="downloads/cartwheel-nih-sep2007-short.pdf">read a recent
grant proposal</a> about some of our software interests.)

<ul>

<li> Neural crest specification in early vertebrate embryogenesis.</li>
<p>
The neural crest is an important developmental tissue that is
specified early on in embryogenesis.  We seek to understand the
molecular events underlying this specification, using both molecular
and embryological techniques, and integrate it with sequence data to
produce a gene network model for neural crest specification.  (This
is our experimental focus.)
<p>
<li> Animal regulatory genomics.</li>
<p>
Much of early development is "hard wired" in the genome's regulatory
elements.  Finding and analyzing regulatory elements in animal genomes
can be very difficult, given the large size and complexity of these
genomes.  We are building tools to help detect, visualize and analyze
regulatory elements.
</p>
<li> Microbial regulatory genomics.</li>
<p>
Gene regulation is an important part of microbial physiology, yet we
still know relatively little about finding and studying regulatory
elements computationally in microbes.  The smaller size of microbial
genomes makes locating transcription factor binding sites easier, but
validation still requires difficult experiments.  We are particularly
interested in techniques for testing the internal consistency of binding
site predictions.
</p>
<li> Evolutionary sequence signatures.</li>
<p>
As genomes evolve, different selection pressures are brought to bear
upon the various functional elements.  These functional elements can
often be recognized computationally from the signature left by
evolution.  We are interested in using such signatures to
computationally find novel genes and regulatory elements.
</p>
<li> Metagenomics.</li>
<p>
Most (99% or more) of microbes cannot be cultured in the lab, yet many
microbes play important environmental or medical roles.  Metagenomics
seeks to understand microbial communities by making use of new
sequencing technologies to sequence unculturable organisms, yet this
data cannot be easily grokked.  We are interested in computational
ways to analyze and understand metagenomic data.
</p>
<li> Software engineering methodologies.</li>
<p>
A critical component of the lab's work will be digesting and
integrating data from multiple sources, both local (e.g. lab data,
collaborator data) and remote (NCBI, ENSEMBL, UCSC, other genome
databases).  This research depends on extensive re-use and maintenance
of old software tools as well as development of new software tools,
a traditional challenge of software engineering.
</p>
</ul>

Studying many topics means many colleagues and many collaborators.
Come by and <A href="contact.html">chat</a> today!

<!--#include file="_bodyfoot.html" -->
