<!--#include file="_header.html" -->
<!--#include file="_bodyhead.html" -->

<div id="container">
<div id="content">

<p>

<h2>Title: A Reference-Free Algorithm for Computational Normalization of Shotgun Sequencing Data</h2>
<p>
C. Titus Brown, Adina Howe, Qingpeng Zhang, Alexis B. Pyrkosz,
and Timothy H. Brom
<p>
<A href='http://arxiv.org/abs/1203.4802'>arXiv preprint</a>
<p>
<blockquote>
Deep shotgun sequencing and analysis of genomes, transcriptomes,
amplified single-cell genomes, and metagenomes has enabled
investigation of a wide range of organisms and ecosystems.  However,
sampling variation in short-read data sets and high sequencing
error rates of modern sequencers present many new computational
challenges in data interpretation.  These challenges have led to the
development of new classes of mapping tools and <i>de novo</i>
assemblers.  These algorithms are challenged by the
continued improvement in sequencing throughput.  We here describe
digital normalization, a single-pass computational algorithm that
systematizes coverage in shotgun sequencing data sets, thereby
decreasing sampling variation, discarding redundant data, and removing
the majority of errors.  Digital normalization substantially reduces
the size of shotgun data sets and decreases the memory and time
requirements for <i>de novo</i> sequence assembly, all without
significantly impacting content of the generated contigs.  We apply
digital normalization to the assembly of microbial genomic data,
amplified single-cell genomic data, and transcriptomic data.  Our
implementation is freely available for use and modification.
</blockquote>

Online resources and data:
<ul>
<li> <a href='http://ged.msu.edu/angus/diginorm-2012/tutorial.html'>A tutorial for running khmer on microbial genomes and eukaryotic transcriptomes</a>.
<p>
<li> Git repository for khmer: <a href='https://github.com/ged-lab/khmer/tree/2012-paper-diginorm'>github.com/ged-lab/khmer/tree/2012-paper-diginorm</a>
<p>
<li> Git repository for paper & data analysis pipeline: <A href='https://github.com/ged-lab/2012-paper-diginorm'>github.com/ged-lab/2012-paper-diginorm</a>
<p>
<li> <a href='http://ged.msu.edu/angus/diginorm-2012/pipeline-notes.html'>Instructions on running the paper analysis pipeline & reproducing the paper</a>
<p>
<li> <a href='notebook/'>HTML view</a> of the <a href='http://ipython.org/ipython-doc/stable/whatsnew/version0.12.html'>ipython notebook</a> containing code and scripts to reproduce the figures in the paper.  (See the <a href='http://ged.msu.edu/angus/diginorm-2012/pipeline-notes.html'>pipeline notes</a> for a runnable version.)
<p>
<li> <a href='https://s3.amazonaws.com/public.ged.msu.edu/2012-paper-diginorm/pipeline-data.tar.gz'>Data required to run the pipeline</a> (.tar.gz, 7.9gb)
<p>
<li> <a href='https://s3.amazonaws.com/public.ged.msu.edu/2012-paper-diginorm/assembly-data.tar.gz'>Assembled microbial genomes and eukaryotic transcriptomes</a> (.tar.gz, 110 mb)
</ul>
</div>
</div>

<!--#include file="_bodyfoot.html" -->

