<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.4: http://docutils.sourceforge.net/" />
<title>Metagenomics (wk 7)</title>
<style type="text/css">

/*
:Author: David Goodger
:Contact: goodger@users.sourceforge.net
:Date: $Date: 2005-12-18 01:56:14 +0100 (Sun, 18 Dec 2005) $
:Revision: $Revision: 4224 $
:Copyright: This stylesheet has been placed in the public domain.

Default cascading style sheet for the HTML output of Docutils.

See http://docutils.sf.net/docs/howto/html-stylesheets.html for how to
customize this style sheet.
*/

/* used to remove borders from tables and images */
.borderless, table.borderless td, table.borderless th {
  border: 0 }

table.borderless td, table.borderless th {
  /* Override padding for "table.docutils td" with "! important".
     The right padding separates the table cells. */
  padding: 0 0.5em 0 0 ! important }

.first {
  /* Override more specific margin styles with "! important". */
  margin-top: 0 ! important }

.last, .with-subtitle {
  margin-bottom: 0 ! important }

.hidden {
  display: none }

a.toc-backref {
  text-decoration: none ;
  color: black }

blockquote.epigraph {
  margin: 2em 5em ; }

dl.docutils dd {
  margin-bottom: 0.5em }

/* Uncomment (and remove this text!) to get bold-faced definition list terms
dl.docutils dt {
  font-weight: bold }
*/

div.abstract {
  margin: 2em 5em }

div.abstract p.topic-title {
  font-weight: bold ;
  text-align: center }

div.admonition, div.attention, div.caution, div.danger, div.error,
div.hint, div.important, div.note, div.tip, div.warning {
  margin: 2em ;
  border: medium outset ;
  padding: 1em }

div.admonition p.admonition-title, div.hint p.admonition-title,
div.important p.admonition-title, div.note p.admonition-title,
div.tip p.admonition-title {
  font-weight: bold ;
  font-family: sans-serif }

div.attention p.admonition-title, div.caution p.admonition-title,
div.danger p.admonition-title, div.error p.admonition-title,
div.warning p.admonition-title {
  color: red ;
  font-weight: bold ;
  font-family: sans-serif }

/* Uncomment (and remove this text!) to get reduced vertical space in
   compound paragraphs.
div.compound .compound-first, div.compound .compound-middle {
  margin-bottom: 0.5em }

div.compound .compound-last, div.compound .compound-middle {
  margin-top: 0.5em }
*/

div.dedication {
  margin: 2em 5em ;
  text-align: center ;
  font-style: italic }

div.dedication p.topic-title {
  font-weight: bold ;
  font-style: normal }

div.figure {
  margin-left: 2em ;
  margin-right: 2em }

div.footer, div.header {
  clear: both;
  font-size: smaller }

div.line-block {
  display: block ;
  margin-top: 1em ;
  margin-bottom: 1em }

div.line-block div.line-block {
  margin-top: 0 ;
  margin-bottom: 0 ;
  margin-left: 1.5em }

div.sidebar {
  margin-left: 1em ;
  border: medium outset ;
  padding: 1em ;
  background-color: #ffffee ;
  width: 40% ;
  float: right ;
  clear: right }

div.sidebar p.rubric {
  font-family: sans-serif ;
  font-size: medium }

div.system-messages {
  margin: 5em }

div.system-messages h1 {
  color: red }

div.system-message {
  border: medium outset ;
  padding: 1em }

div.system-message p.system-message-title {
  color: red ;
  font-weight: bold }

div.topic {
  margin: 2em }

h1.section-subtitle, h2.section-subtitle, h3.section-subtitle,
h4.section-subtitle, h5.section-subtitle, h6.section-subtitle {
  margin-top: 0.4em }

h1.title {
  text-align: center }

h2.subtitle {
  text-align: center }

hr.docutils {
  width: 75% }

img.align-left {
  clear: left }

img.align-right {
  clear: right }

ol.simple, ul.simple {
  margin-bottom: 1em }

ol.arabic {
  list-style: decimal }

ol.loweralpha {
  list-style: lower-alpha }

ol.upperalpha {
  list-style: upper-alpha }

ol.lowerroman {
  list-style: lower-roman }

ol.upperroman {
  list-style: upper-roman }

p.attribution {
  text-align: right ;
  margin-left: 50% }

p.caption {
  font-style: italic }

p.credits {
  font-style: italic ;
  font-size: smaller }

p.label {
  white-space: nowrap }

p.rubric {
  font-weight: bold ;
  font-size: larger ;
  color: maroon ;
  text-align: center }

p.sidebar-title {
  font-family: sans-serif ;
  font-weight: bold ;
  font-size: larger }

p.sidebar-subtitle {
  font-family: sans-serif ;
  font-weight: bold }

p.topic-title {
  font-weight: bold }

pre.address {
  margin-bottom: 0 ;
  margin-top: 0 ;
  font-family: serif ;
  font-size: 100% }

pre.literal-block, pre.doctest-block {
  margin-left: 2em ;
  margin-right: 2em ;
  background-color: #eeeeee }

span.classifier {
  font-family: sans-serif ;
  font-style: oblique }

span.classifier-delimiter {
  font-family: sans-serif ;
  font-weight: bold }

span.interpreted {
  font-family: sans-serif }

span.option {
  white-space: nowrap }

span.pre {
  white-space: pre }

span.problematic {
  color: red }

span.section-subtitle {
  /* font-size relative to parent (h1..h6 element) */
  font-size: 80% }

table.citation {
  border-left: solid 1px gray;
  margin-left: 1px }

table.docinfo {
  margin: 2em 4em }

table.docutils {
  margin-top: 0.5em ;
  margin-bottom: 0.5em }

table.footnote {
  border-left: solid 1px black;
  margin-left: 1px }

table.docutils td, table.docutils th,
table.docinfo td, table.docinfo th {
  padding-left: 0.5em ;
  padding-right: 0.5em ;
  vertical-align: top }

table.docutils th.field-name, table.docinfo th.docinfo-name {
  font-weight: bold ;
  text-align: left ;
  white-space: nowrap ;
  padding-left: 0 }

h1 tt.docutils, h2 tt.docutils, h3 tt.docutils,
h4 tt.docutils, h5 tt.docutils, h6 tt.docutils {
  font-size: 100% }

tt.docutils {
  background-color: #eeeeee }

ul.auto-toc {
  list-style-type: none }

</style>
</head>
<body>
<div class="document" id="metagenomics-wk-7">
<h1 class="title">Metagenomics (wk 7)</h1>
<p>Feb 26th, 2009</p>
<p>CSE 891</p>
<p>---</p>
<!-- nat review on metagenomics? -->
<!-- nature gordon lab on lean & obese twins -->
<div class="section">
<h1><a id="metagenomics-big-issues" name="metagenomics-big-issues">Metagenomics -- big issues</a></h1>
<p>Population</p>
<p>(Neutral) evolution &amp; sequence swapping</p>
<dl class="docutils">
<dt>What is in genomes may not be expressed</dt>
<dd>(--&gt; metatranscriptomics)</dd>
</dl>
</div>
<div class="section">
<h1><a id="kinds-of-analyses" name="kinds-of-analyses">Kinds of analyses</a></h1>
<p>So, someone hands you 100,000 sequences that are 450 bases in length.
Now what?</p>
<p>(Note, more data generated from metagenomics than for all sequenced
bacterial genomes, as of 2007.  Now, there's even more.)</p>
<p>Our original questions: diversity and (functional) content.</p>
<p>Assembly:</p>
<blockquote>
<ul>
<li><p class="first">diversity can be immense; little or no idea of how much sequencing
is required</p>
</li>
<li><p class="first">high polymorphism within population (much worse than haplotype problem
in animals!)</p>
</li>
<li><p class="first">one sequencing approach: fosmid (large plasmid) creation, then
sequencing of each fosmid (think pooled BAC approach, as in sea
urchin)</p>
</li>
<li><dl class="first docutils">
<dt>can try to &quot;paint&quot; onto nearby genomes by homology search.</dt>
<dd><ul class="first last simple">
<li>but what's a nearby genome?</li>
<li>what are you missing?</li>
<li>is this reliable?</li>
</ul>
</dd>
</dl>
</li>
</ul>
</blockquote>
<p>Separating species by nt features</p>
<blockquote>
<ul class="simple">
<li>codon and G/C bias in organisms is quite strong, somewhat uniform across
genome, &amp; species dependent</li>
<li>can get &quot;word count&quot; statistics and try to separate sequences into
&quot;bins&quot;</li>
<li>way of estimating # of species (lower bound)</li>
<li>doesn't work very well for short sequences, because signal is too low.</li>
<li>back to assembly...</li>
</ul>
</blockquote>
<p>De novo gene prediction</p>
<blockquote>
<ul class="simple">
<li>it didn't work in animals very well, you may recall; but it worked
ok in bacteria.</li>
<li>now we have shorter traces...</li>
<li>what sequence features did they use in bacteria -- and are they
likely to be there in metagenomic data?
(Need longer contigs)</li>
<li>back to assembly...</li>
</ul>
</blockquote>
<p>Phylogenetic assessment with MEGAN</p>
<blockquote>
<ul class="simple">
<li>map proteins onto known proteins with BLAST</li>
<li>look at top scoring hits &amp; extract taxonomic identity of originating species</li>
<li>map position onto taxonomic tree and take inclusive node as &quot;highest
resolution&quot; phylogenetic assignment</li>
<li>seems to make sense when I look at the data...</li>
<li>...but one big problem... what is it?</li>
</ul>
</blockquote>
</div>
<div class="section">
<h1><a id="dark-sequence" name="dark-sequence">&quot;Dark sequence&quot;</a></h1>
<p>~60-80% of sequence from metagenomics projects doesn't look like any known
genes when BLAST is used.</p>
<p>What is it??</p>
<blockquote>
<ul>
<li><p class="first">new, unknown genes</p>
<p>(implication: huge diversity)</p>
</li>
<li><p class="first">non-genic crud</p>
<p>(that's odd, most bacterial genomes seem to be full of genes... 80%!)</p>
</li>
<li><p class="first">very divergent genes</p>
</li>
</ul>
</blockquote>
<p>Some evidence that they're divergent genes of known function.</p>
</div>
<div class="section">
<h1><a id="functional-annotation" name="functional-annotation">Functional annotation</a></h1>
<p>Briefly -- how do proteins work?  (The 1000 foot view/80% correct)</p>
<p>Strings of functional domains attached with spacers</p>
<blockquote>
<ul class="simple">
<li>the function of proteins is tied to folding structure</li>
<li>protein sequence can often vary substantially and still be functional
(20-30% identity =&gt; same function!)</li>
</ul>
</blockquote>
<p>Homology searches (BLAST) tend to work because rate of sequence change
in evolution is rather slow.  <em>Implication: quite a lot of sequence
conservation in proteins is &quot;accidental&quot;, and does not necessarily
imply function.</em></p>
<p>This also means that BLAST may not be the best way to assign function.</p>
<p>More sensitive searches include HMMER, which essentially uses a
position-specific scoring matrix to look at key important residues.
But it needs much more information up front... take sequence
alignments from multiple organisms, generate PSSM.</p>
<p>Inferring function:</p>
<p>Homology search with BLAST.</p>
<blockquote>
<ul class="simple">
<li>homology: orthology (speciation) vs paralogy (gene duplication)</li>
<li>often substrate specificity for enzymatic reactions is tied to
slight functional divergence in sequence of domains, so you have to
worry about specific thresholds.</li>
</ul>
</blockquote>
<p>(KEGG pathway searches.)</p>
<p>Protein domain search (e.g. PFAM) is a more sensitive way to detect domains.</p>
<p>It's also 1000s of times slower.</p>
<p>Most efforts saw ~30-50% assignment of function to genes
(e.g. Sargasso Sea paper).  Bork paper (2007; assigned reading for Th)
discusses techniques for additional functional assignment: all stuff
you've seen before,</p>
<blockquote>
<ol class="arabic simple">
<li>high similarity to, or close to, known gene of highly specific function</li>
<li>similarity to known gene w/o highly specific function</li>
<li>similarity to known gene of unknown function</li>
<li>no similarity to anything known</li>
</ol>
</blockquote>
<p>They look only at predicted ORFs, so they eliminate lots of crappy data
at the expense of slightly higher false negatives.</p>
<p>Can assign ~80%, comparable to the results of their pipeline on known
genomes.</p>
</div>
<div class="section">
<h1><a id="back-to-the-original-question-s" name="back-to-the-original-question-s">Back to the original question(s)</a></h1>
<p>Assessing diversity.</p>
<p>Assessing content.</p>
<p>Assessing rate of evolution!?</p>
<p>This is a very open field ;).  Most of the interesting stuff is going to
be going on in three areas, IMO:</p>
<blockquote>
<ul class="simple">
<li>functional testing of hypotheses and/or serious correlation assays
between similar microbial environments, using metagenomics</li>
<li>metatranscriptomics</li>
<li>protein domain analysis and other &quot;hard to discover&quot; sequence features
(ncRNA?)</li>
</ul>
</blockquote>
</div>
</div>
</body>
</html>
