<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.4: http://docutils.sourceforge.net/" />
<title>Large scale mRNA expression analysis (wk 13)</title>
<style type="text/css">

/*
:Author: David Goodger
:Contact: goodger@users.sourceforge.net
:Date: $Date: 2005-12-18 01:56:14 +0100 (Sun, 18 Dec 2005) $
:Revision: $Revision: 4224 $
:Copyright: This stylesheet has been placed in the public domain.

Default cascading style sheet for the HTML output of Docutils.

See http://docutils.sf.net/docs/howto/html-stylesheets.html for how to
customize this style sheet.
*/

/* used to remove borders from tables and images */
.borderless, table.borderless td, table.borderless th {
  border: 0 }

table.borderless td, table.borderless th {
  /* Override padding for "table.docutils td" with "! important".
     The right padding separates the table cells. */
  padding: 0 0.5em 0 0 ! important }

.first {
  /* Override more specific margin styles with "! important". */
  margin-top: 0 ! important }

.last, .with-subtitle {
  margin-bottom: 0 ! important }

.hidden {
  display: none }

a.toc-backref {
  text-decoration: none ;
  color: black }

blockquote.epigraph {
  margin: 2em 5em ; }

dl.docutils dd {
  margin-bottom: 0.5em }

/* Uncomment (and remove this text!) to get bold-faced definition list terms
dl.docutils dt {
  font-weight: bold }
*/

div.abstract {
  margin: 2em 5em }

div.abstract p.topic-title {
  font-weight: bold ;
  text-align: center }

div.admonition, div.attention, div.caution, div.danger, div.error,
div.hint, div.important, div.note, div.tip, div.warning {
  margin: 2em ;
  border: medium outset ;
  padding: 1em }

div.admonition p.admonition-title, div.hint p.admonition-title,
div.important p.admonition-title, div.note p.admonition-title,
div.tip p.admonition-title {
  font-weight: bold ;
  font-family: sans-serif }

div.attention p.admonition-title, div.caution p.admonition-title,
div.danger p.admonition-title, div.error p.admonition-title,
div.warning p.admonition-title {
  color: red ;
  font-weight: bold ;
  font-family: sans-serif }

/* Uncomment (and remove this text!) to get reduced vertical space in
   compound paragraphs.
div.compound .compound-first, div.compound .compound-middle {
  margin-bottom: 0.5em }

div.compound .compound-last, div.compound .compound-middle {
  margin-top: 0.5em }
*/

div.dedication {
  margin: 2em 5em ;
  text-align: center ;
  font-style: italic }

div.dedication p.topic-title {
  font-weight: bold ;
  font-style: normal }

div.figure {
  margin-left: 2em ;
  margin-right: 2em }

div.footer, div.header {
  clear: both;
  font-size: smaller }

div.line-block {
  display: block ;
  margin-top: 1em ;
  margin-bottom: 1em }

div.line-block div.line-block {
  margin-top: 0 ;
  margin-bottom: 0 ;
  margin-left: 1.5em }

div.sidebar {
  margin-left: 1em ;
  border: medium outset ;
  padding: 1em ;
  background-color: #ffffee ;
  width: 40% ;
  float: right ;
  clear: right }

div.sidebar p.rubric {
  font-family: sans-serif ;
  font-size: medium }

div.system-messages {
  margin: 5em }

div.system-messages h1 {
  color: red }

div.system-message {
  border: medium outset ;
  padding: 1em }

div.system-message p.system-message-title {
  color: red ;
  font-weight: bold }

div.topic {
  margin: 2em }

h1.section-subtitle, h2.section-subtitle, h3.section-subtitle,
h4.section-subtitle, h5.section-subtitle, h6.section-subtitle {
  margin-top: 0.4em }

h1.title {
  text-align: center }

h2.subtitle {
  text-align: center }

hr.docutils {
  width: 75% }

img.align-left {
  clear: left }

img.align-right {
  clear: right }

ol.simple, ul.simple {
  margin-bottom: 1em }

ol.arabic {
  list-style: decimal }

ol.loweralpha {
  list-style: lower-alpha }

ol.upperalpha {
  list-style: upper-alpha }

ol.lowerroman {
  list-style: lower-roman }

ol.upperroman {
  list-style: upper-roman }

p.attribution {
  text-align: right ;
  margin-left: 50% }

p.caption {
  font-style: italic }

p.credits {
  font-style: italic ;
  font-size: smaller }

p.label {
  white-space: nowrap }

p.rubric {
  font-weight: bold ;
  font-size: larger ;
  color: maroon ;
  text-align: center }

p.sidebar-title {
  font-family: sans-serif ;
  font-weight: bold ;
  font-size: larger }

p.sidebar-subtitle {
  font-family: sans-serif ;
  font-weight: bold }

p.topic-title {
  font-weight: bold }

pre.address {
  margin-bottom: 0 ;
  margin-top: 0 ;
  font-family: serif ;
  font-size: 100% }

pre.literal-block, pre.doctest-block {
  margin-left: 2em ;
  margin-right: 2em ;
  background-color: #eeeeee }

span.classifier {
  font-family: sans-serif ;
  font-style: oblique }

span.classifier-delimiter {
  font-family: sans-serif ;
  font-weight: bold }

span.interpreted {
  font-family: sans-serif }

span.option {
  white-space: nowrap }

span.pre {
  white-space: pre }

span.problematic {
  color: red }

span.section-subtitle {
  /* font-size relative to parent (h1..h6 element) */
  font-size: 80% }

table.citation {
  border-left: solid 1px gray;
  margin-left: 1px }

table.docinfo {
  margin: 2em 4em }

table.docutils {
  margin-top: 0.5em ;
  margin-bottom: 0.5em }

table.footnote {
  border-left: solid 1px black;
  margin-left: 1px }

table.docutils td, table.docutils th,
table.docinfo td, table.docinfo th {
  padding-left: 0.5em ;
  padding-right: 0.5em ;
  vertical-align: top }

table.docutils th.field-name, table.docinfo th.docinfo-name {
  font-weight: bold ;
  text-align: left ;
  white-space: nowrap ;
  padding-left: 0 }

h1 tt.docutils, h2 tt.docutils, h3 tt.docutils,
h4 tt.docutils, h5 tt.docutils, h6 tt.docutils {
  font-size: 100% }

tt.docutils {
  background-color: #eeeeee }

ul.auto-toc {
  list-style-type: none }

</style>
</head>
<body>
<div class="document" id="large-scale-mrna-expression-analysis-wk-13">
<h1 class="title">Large scale mRNA expression analysis (wk 13)</h1>
<p>April 9th, 2009</p>
<p>CSE 891</p>
<p>---</p>
<p>Measure mRNA on a whole-cell/whole-tissue basis: time courses,
different conditions, various perturbations.</p>
<div class="section">
<h1><a id="techniques" name="techniques">Techniques</a></h1>
<p>Microarrays:</p>
<blockquote>
<ul class="simple">
<li>oligo arrays</li>
<li>EST arrays</li>
<li>oligo tiling arrays</li>
</ul>
</blockquote>
<p>SAGE (&quot;Serial Analysis of Gene Expression&quot;)</p>
<p>Next gen sequencing: DGE, RNAseq.</p>
<dl class="docutils">
<dt>Issues:</dt>
<dd><ul class="first last simple">
<li>prior knowledge, ex-post-facto knowledge</li>
<li>splice isoforms</li>
<li>dynamic range</li>
<li>sensitivity &amp; amount of input material</li>
</ul>
</dd>
</dl>
</div>
<div class="section">
<h1><a id="what-are-you-trying-to-measure" name="what-are-you-trying-to-measure">What are you trying to measure?</a></h1>
<p>mRNA levels are relatively easy to measure, ...</p>
<p>...but quite a bit of mRNA isn't functional until it is translated.</p>
<p>mRNA level &lt;-&gt; protein level correspondence?  (Ming will talk about
this on Tuesday.)</p>
</div>
<div class="section">
<h1><a id="what-s-the-point" name="what-s-the-point">What's the point?</a></h1>
<p>(First, digest into numbers.)</p>
<p>Clustering/covariance, &quot;guilt by association&quot;.  What genes change together?</p>
<p>Gene discovery for pathways.</p>
<p>Looking for regulatory mechanisms (e.g. t.f. binding sites.)</p>
</div>
<div class="section">
<h1><a id="basic-data-processing-issues" name="basic-data-processing-issues">Basic data processing issues</a></h1>
<p>Standardization; relative levels.  Absolute vs relative measurements.</p>
<p>Normalizing: shifting the distribution to fit some standard (Is this
the same as standardization?)</p>
<p>Converting to rank vs using the numbers straight.   (Noise control method?)</p>
<p>Significance/p-value calculation??</p>
</div>
<div class="section">
<h1><a id="identifying-genes-and-functions" name="identifying-genes-and-functions">Identifying genes and functions</a></h1>
<p>&gt; 20,000 genes in an animal; how many of them do you know by name?</p>
<p>(Microbiologists have a slightly easier time of it - ~1-3k genes.)</p>
<p>Annotations on genes -- human readable, or machine readable, or both?</p>
<p>How do you convert?  What are the uses?</p>
<p>&quot;Controlled vocabulary.&quot; -- unambiguous terminology.</p>
<p>Gene Ontology:</p>
<pre class="literal-block">
[Term]
id: GO:0016021
name: integral to membrane
namespace: cellular_component
def: &quot;Penetrating at least one phospholipid bilayer of a membrane.
   May also refer to the state of being buried in the bilayer with
   no exposure outside the bilayer. When used to describe a protein,
   indicates that all or part of the peptide sequence is embedded in
   the membrane.&quot; [GOC:go_curators]
subset: gosubset_prok
synonym: &quot;transmembrane&quot; EXACT []
xref: Wikipedia:Transmembrane_protein
is_a: GO:0031224 ! intrinsic to membrane
</pre>
<p>Three &quot;namespaces&quot;: biological processes, molecular functions, and
cellular components.</p>
<p>Arranged in a &quot;Directed Acyclic Graph&quot;, or DAG - each node has one or
more parents, no &quot;cycles&quot; (node cannot be ancestor of itelf).  This
makes it easy to computationally &quot;digest&quot; data and provide both
shallow/broad and deep/specific views of gene expression.</p>
<p>Individual genome consortia provide mappings between proteins and GO
IDs.</p>
<p>Issues:</p>
<blockquote>
<ul>
<li><p class="first">assignment of GO function based on homology/orthology.  (...
and paralogy/orthology issues)</p>
</li>
<li><p class="first">who makes the GO annotations, and what are they based on?</p>
<blockquote>
<p>&quot;Inferred from Electronic Annotation&quot;</p>
</blockquote>
<p>vs</p>
<blockquote>
<p>&quot;Inferred from Experiment&quot;</p>
</blockquote>
<p>vs</p>
<blockquote>
<p>&quot;Non-traceable author statement&quot;</p>
</blockquote>
</li>
<li><p class="first">transitivity of annotations.  To quote Erich Schwarz, Wormbase:</p>
</li>
</ul>
<blockquote>
&quot;&quot;&quot;
[For IEA], ... you want to make sure that these transitive
inferences are not, themselves, done from terms whose status is IEA
or ISS.  Otherwise, you get into an infinite regress of GO
annotation by similarity of X to Y, which in turn was annotated by
similarity to Z, and Z by similarity to Theta, and ...  eventually
you have a urochordate gene being annotated by a five-steps removed
similarity to an <em>actually observed</em> function in Bacillus subtilis.
And the B. subtilis paper describing this function was 20 years ago.
And the first author was sort of delusional.  That sort of botch-up
can really put the OHMygod into genOHMics, if you know what I mean.
&quot;&quot;&quot;</blockquote>
</blockquote>
<p>Any guesses as to where the bulk of GO annotations come from?</p>
<p>--</p>
<p>Open ended question: what kind of annotations do we want on genes?
Human or &quot;machine&quot; readable?</p>
</div>
</div>
</body>
</html>
