<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.4: http://docutils.sourceforge.net/" />
<title>Next-generation Sequencing</title>
<style type="text/css">

/*
:Author: David Goodger
:Contact: goodger@users.sourceforge.net
:Date: $Date: 2005-12-18 01:56:14 +0100 (Sun, 18 Dec 2005) $
:Revision: $Revision: 4224 $
:Copyright: This stylesheet has been placed in the public domain.

Default cascading style sheet for the HTML output of Docutils.

See http://docutils.sf.net/docs/howto/html-stylesheets.html for how to
customize this style sheet.
*/

/* used to remove borders from tables and images */
.borderless, table.borderless td, table.borderless th {
  border: 0 }

table.borderless td, table.borderless th {
  /* Override padding for "table.docutils td" with "! important".
     The right padding separates the table cells. */
  padding: 0 0.5em 0 0 ! important }

.first {
  /* Override more specific margin styles with "! important". */
  margin-top: 0 ! important }

.last, .with-subtitle {
  margin-bottom: 0 ! important }

.hidden {
  display: none }

a.toc-backref {
  text-decoration: none ;
  color: black }

blockquote.epigraph {
  margin: 2em 5em ; }

dl.docutils dd {
  margin-bottom: 0.5em }

/* Uncomment (and remove this text!) to get bold-faced definition list terms
dl.docutils dt {
  font-weight: bold }
*/

div.abstract {
  margin: 2em 5em }

div.abstract p.topic-title {
  font-weight: bold ;
  text-align: center }

div.admonition, div.attention, div.caution, div.danger, div.error,
div.hint, div.important, div.note, div.tip, div.warning {
  margin: 2em ;
  border: medium outset ;
  padding: 1em }

div.admonition p.admonition-title, div.hint p.admonition-title,
div.important p.admonition-title, div.note p.admonition-title,
div.tip p.admonition-title {
  font-weight: bold ;
  font-family: sans-serif }

div.attention p.admonition-title, div.caution p.admonition-title,
div.danger p.admonition-title, div.error p.admonition-title,
div.warning p.admonition-title {
  color: red ;
  font-weight: bold ;
  font-family: sans-serif }

/* Uncomment (and remove this text!) to get reduced vertical space in
   compound paragraphs.
div.compound .compound-first, div.compound .compound-middle {
  margin-bottom: 0.5em }

div.compound .compound-last, div.compound .compound-middle {
  margin-top: 0.5em }
*/

div.dedication {
  margin: 2em 5em ;
  text-align: center ;
  font-style: italic }

div.dedication p.topic-title {
  font-weight: bold ;
  font-style: normal }

div.figure {
  margin-left: 2em ;
  margin-right: 2em }

div.footer, div.header {
  clear: both;
  font-size: smaller }

div.line-block {
  display: block ;
  margin-top: 1em ;
  margin-bottom: 1em }

div.line-block div.line-block {
  margin-top: 0 ;
  margin-bottom: 0 ;
  margin-left: 1.5em }

div.sidebar {
  margin-left: 1em ;
  border: medium outset ;
  padding: 1em ;
  background-color: #ffffee ;
  width: 40% ;
  float: right ;
  clear: right }

div.sidebar p.rubric {
  font-family: sans-serif ;
  font-size: medium }

div.system-messages {
  margin: 5em }

div.system-messages h1 {
  color: red }

div.system-message {
  border: medium outset ;
  padding: 1em }

div.system-message p.system-message-title {
  color: red ;
  font-weight: bold }

div.topic {
  margin: 2em }

h1.section-subtitle, h2.section-subtitle, h3.section-subtitle,
h4.section-subtitle, h5.section-subtitle, h6.section-subtitle {
  margin-top: 0.4em }

h1.title {
  text-align: center }

h2.subtitle {
  text-align: center }

hr.docutils {
  width: 75% }

img.align-left {
  clear: left }

img.align-right {
  clear: right }

ol.simple, ul.simple {
  margin-bottom: 1em }

ol.arabic {
  list-style: decimal }

ol.loweralpha {
  list-style: lower-alpha }

ol.upperalpha {
  list-style: upper-alpha }

ol.lowerroman {
  list-style: lower-roman }

ol.upperroman {
  list-style: upper-roman }

p.attribution {
  text-align: right ;
  margin-left: 50% }

p.caption {
  font-style: italic }

p.credits {
  font-style: italic ;
  font-size: smaller }

p.label {
  white-space: nowrap }

p.rubric {
  font-weight: bold ;
  font-size: larger ;
  color: maroon ;
  text-align: center }

p.sidebar-title {
  font-family: sans-serif ;
  font-weight: bold ;
  font-size: larger }

p.sidebar-subtitle {
  font-family: sans-serif ;
  font-weight: bold }

p.topic-title {
  font-weight: bold }

pre.address {
  margin-bottom: 0 ;
  margin-top: 0 ;
  font-family: serif ;
  font-size: 100% }

pre.literal-block, pre.doctest-block {
  margin-left: 2em ;
  margin-right: 2em ;
  background-color: #eeeeee }

span.classifier {
  font-family: sans-serif ;
  font-style: oblique }

span.classifier-delimiter {
  font-family: sans-serif ;
  font-weight: bold }

span.interpreted {
  font-family: sans-serif }

span.option {
  white-space: nowrap }

span.pre {
  white-space: pre }

span.problematic {
  color: red }

span.section-subtitle {
  /* font-size relative to parent (h1..h6 element) */
  font-size: 80% }

table.citation {
  border-left: solid 1px gray;
  margin-left: 1px }

table.docinfo {
  margin: 2em 4em }

table.docutils {
  margin-top: 0.5em ;
  margin-bottom: 0.5em }

table.footnote {
  border-left: solid 1px black;
  margin-left: 1px }

table.docutils td, table.docutils th,
table.docinfo td, table.docinfo th {
  padding-left: 0.5em ;
  padding-right: 0.5em ;
  vertical-align: top }

table.docutils th.field-name, table.docinfo th.docinfo-name {
  font-weight: bold ;
  text-align: left ;
  white-space: nowrap ;
  padding-left: 0 }

h1 tt.docutils, h2 tt.docutils, h3 tt.docutils,
h4 tt.docutils, h5 tt.docutils, h6 tt.docutils {
  font-size: 100% }

tt.docutils {
  background-color: #eeeeee }

ul.auto-toc {
  list-style-type: none }

</style>
</head>
<body>
<div class="document" id="next-generation-sequencing">
<h1 class="title">Next-generation Sequencing</h1>
<p>Feb 12, 2009</p>
<p>CSE 891</p>
<p>--</p>
<p>Sanger sequencing - cloning, growing, etc.</p>
<p>Bias issues: going thru bacteria.</p>
<p>Scales poorly.</p>
<p>Read length ~900 bp</p>
<p>Used for most sequencing to date, still used for nasty whole-genome
sequencing.</p>
<div class="section">
<h1><a id="new-technologies" name="new-technologies">New technologies</a></h1>
<p>New technologies emerge over the last few years.</p>
<p>(Probably one of the coolest things to happen in biotech in the last
few years)</p>
<p>One basic strategy (454, Solexa, Polonator)</p>
<blockquote>
<ul class="simple">
<li>Isolate individual DNA molecules</li>
<li>Use PCR to generate &quot;PCR colonies&quot;, or polonies</li>
<li>Replicate one more time, but with fluorescent nucleotides</li>
<li>Image at each nucleotide addition</li>
</ul>
</blockquote>
<p>Note: PCR, and PCR bias.</p>
<div class="section">
<h2><a id="id1" name="id1">454</a></h2>
<p>Most mature technology, along with Solexa.</p>
<p>Prone to errors in reading long homopolymer runs.</p>
<p>Paired-end sequencing: one strategy is circularization, selection,
digestion, sequencing from new ligated pairs.</p>
<p>Titanium read stats: 500 bp mode, 400k (?) reads</p>
<p>~$10k</p>
</div>
<div class="section">
<h2><a id="solexa-illumina" name="solexa-illumina">Solexa(/Illumina)</a></h2>
<p>Very mature.</p>
<p>Immobilization and growth on solid substrate.</p>
<p>25-40m reads, 25-40 bp.  Soon up to 70bp?</p>
<p>Paired-end is readily available.</p>
<p>~$10k</p>
</div>
<div class="section">
<h2><a id="solid" name="solid">SOLiD</a></h2>
<p>Ligation-based sequencing</p>
<p>Construct oligos with mostly &quot;wild cards&quot; and all possible 2-mers in
the middle</p>
<pre class="literal-block">
NNNNNNNNNNN-AA-NNNNNNNNNNN
NNNNNNNNNNN-AT-NNNNNNNNNNN
NNNNNNNNNNN-AC-NNNNNNNNNNN
NNNNNNNNNNN-AG-NNNNNNNNNNN
NNNNNNNNNNN-TA-NNNNNNNNNNN
</pre>
<p>and label each one with a different fluorescent tag.  Ligate.  The tag
that shows up in a particular polony tells you what sequence is in the
middle.</p>
<p>10x Solexa (?)</p>
</div>
<div class="section">
<h2><a id="helicos" name="helicos">Helicos</a></h2>
<p>Not polony based -- single-molecule sequencing.</p>
<p>poly-A tail immobilization followed by fluorescent &quot;quad&quot; incorporation &amp;
imaging.</p>
<p>25-90 megabase/hr according to PR</p>
</div>
<div class="section">
<h2><a id="polonator" name="polonator">Polonator</a></h2>
<p>&quot;Open source&quot; polony machine: open hardware, open software. You pay for
the hardware and get full instructions on modification.</p>
<p>Currently mainly for biochemistry enthusiasts :)</p>
</div>
<div class="section">
<h2><a id="pacific-biosciences" name="pacific-biosciences">Pacific Biosciences</a></h2>
<p>Not polony based -- single molecule sequencing.</p>
<p>Image actual polymerase putting tags on.  Not in production yet, but
promises to deliver ~1-10kb reads.</p>
<p>The only sequencing technique that may not have nonsense-read problems.</p>
</div>
</div>
<div class="section">
<h1><a id="what-use-are-these-techniques" name="what-use-are-these-techniques">What use are these techniques?</a></h1>
<p>Cheap!</p>
<p>Massively parallel.</p>
<p>Easily (?) scalable.</p>
<p>Many, many simultaneous samples.</p>
<div class="section">
<h2><a id="but" name="but">...but?</a></h2>
<p>Shorter reads (tho improving).</p>
<p>Error models &amp; biases not well understood.</p>
<p>Substantial number of &quot;error&quot; reads (mixed polonies, etc.?)</p>
<p>Can now generate more data than can easily be analyzed.</p>
</div>
<div class="section">
<h2><a id="uses" name="uses">Uses</a></h2>
<p>Re-sequencing: looking for SNPs, copy number variation, indels, etc.</p>
<p>Large-scale RNA sequencing</p>
<p>Counting molecules: useful for</p>
<blockquote>
<ul class="simple">
<li>expression assay</li>
<li>ChIP assay</li>
<li>overlap assay (?)</li>
</ul>
</blockquote>
<p>Sequencing &quot;from the wild&quot;, or environmental sequencing</p>
<p>Genome sequencing?</p>
<p>--</p>
<p>Thinking about use &amp; features:</p>
<blockquote>
<ul class="simple">
<li>for indels (insertion-deletion analysis), want long reads OR paired-end
reads</li>
<li>for de novo sequencing, longer reads are better: otherwise no way to
identify sequence, because you can't map it to known genome.</li>
<li>for <em>counting</em>, as long as reads long enough to be mappable, want
as many samples as possible.  (Think stddev.)</li>
</ul>
</blockquote>
<p>Computational analysis of all this data is still a tricky business.
(...because algorithms seldom scale linearly with the amount of data to
be analyzed.)</p>
<p>So is storage -- quote from ABI rep, paraphrased: &quot;we discard the
raw image data after initial processing into sequence, because storing
the raw data is more expensive than re-running the sample.&quot;</p>
</div>
<div class="section">
<h2><a id="analysis" name="analysis">Analysis</a></h2>
<p>Assembly is a bitch.</p>
<p>Mapping can be a problem for short reads, too.  ~25 bp reads:</p>
<blockquote>
<ul class="simple">
<li>basic problem is that in (non-random) genomes, ~25 bp hits many times;
what to do with those tags?</li>
<li>cannot map across even tiny repetitive elements w/o paired end</li>
<li>&quot;nonsense&quot; reads confuse the situation: if 40% of the reads aren't
mappable...??</li>
</ul>
</blockquote>
<p>Not a great diversity of mapping &amp; analysis software out there yet.</p>
<p>Vast amounts of data are difficult for biologists to deal with, and
complicate things for computational people too.</p>
</div>
</div>
</div>
</body>
</html>
